ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/DC1-L2LEAF1A.yml

```diff
@@ -2,3 +2,8 @@
 # Test that having underlay_ipv6 will not require
 # loopback_ipv6_pool for devices that are not underlay_router
 underlay_ipv6: true
+underlay_rfc5549: true
```

Comment:
This line is not in the cherry_picked PR, but maybe this actually reveals another issue with l2 leaf and underlay_rfc5549, since you see the ipv6_unicast_routing and ip_routing_ipv6_interfaces being set.

Reply:
Removed the line.